sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/CloudPoolIntegrationTests.cs

```diff
@@ -23,12 +23,11 @@ namespace BatchClientIntegrationTests
     using Xunit.Abstractions;
     using Protocol = Microsoft.Azure.Batch.Protocol;
     using Microsoft.Azure.Batch.Integration.Tests.IntegrationTestUtilities;
-    using System.Net.PeerToPeer;
 
     public class CloudPoolIntegrationTests
     {
         private readonly ITestOutputHelper testOutputHelper;
-        private static readonly TimeSpan LongTestTimeout = TimeSpan.FromMinutes(5);
+        private static readonly TimeSpan LongTestTimeout = TimeSpan.FromMinutes(15);
```

Comment:
Why this increase?

Reply:
I shouldn't wait for 1 min in each pool state check, revert to 5 mins in LongTestTimeout and did some test code changes.